common/web/base-web/index.d.ts

```diff
@@ -46,6 +46,15 @@ declare module '*.svg' {
   export default value;
 }
 
+declare module '*.mp4' {
+  const value: string;
+  export default value;
+}
+
+declare module '*.m4a' {
+  const value: string;
+  export default value
```

Comment:
```suggestion
  export default value;
```

Reply:
Heh - was a bad idea to try to use GitHub to edit the code 😆

Reply:
I'm still not used to the multi-line suggestions 😅